cms/templates/base.html

```diff
@@ -85,7 +85,7 @@
                       url_name: "${context_course.location.name | h}",
                       org: "${context_course.location.org | h}",
                       num: "${context_course.location.course | h}",
-                      display_course_number: "${_(context_course.display_coursenumber)}",
+                      display_course_number: "${_(context_course.display_coursenumber) if context_course.display_coursenumber else ''}",
```

Comment:
@adampalay @symbolist One thing I didn't noticed at first, because it didn't change in this particular commit, was the introduction of `_()` here.  I'm guessing this shouldn't be translated.  Can you confirm?